Documention of secrets param incorrect?

### TL;DR

The `secrets` param format doesn't seem to match the documentation?

### Expected behavior

Expected this format to work:

```
 secrets: |-
    # As an environment variable:
    KEY1=secret-key-1:latest
```

But it seems that only the old format works currently?

```
 secrets: |-
   KEY1=projects/p/secrets/s/versions/v
```

### Observed behavior

Error: google-github-actions/deploy-cloud-functions failed with: failed to parse secret reference "ASTRAL_LOCATION:latest": unknown format. Secrets should be of the format "projects/p/secrets/s/versions/v".

### Action YAML

```yaml
name: Deploy
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
jobs:
  deploy:
    runs-on: ubuntu-latest
    permissions:
      contents: 'read'
      id-token: 'write'
    concurrency: "prod"
    steps:
      - uses: 'actions/checkout@v4.1.7'
      - id: 'auth'
        name: 'Authenticate to Google Cloud'
        uses: 'google-github-actions/auth@v2.1.5'
        with:
          workload_identity_provider: ${{ vars.GCP_WORKLOAD_IDENTITY_PROVIDER }}
          service_account: ${{ vars.GCP_SERVICE_ACCOUNT }}
      - uses: actions/setup-python@v5.2.0
        with:
          python-version: 3.11
      - name: Install poetry
        uses: abatilo/actions-poetry@v3.0.0
      - name: Install dependencies
        run: poetry install
      - name: Test
        run: poetry run pytest -v
      - name: Generate Requirements
        run: poetry export --output=requirements.txt
      - id: 'deploy'
        uses: 'google-github-actions/deploy-cloud-functions@v3.0.5'
        with:
          name: ${{ vars.GCP_CLOUD_FUNCTION_NAME }}
          runtime: 'python311'
          environment: GEN_2
          max_instance_count: 2
          entry_point: "update"
          # TODO put these all in github or all in gcp
          environment_variables: |-
            DEVICE_ID=${{ vars.DEVICE_ID}}
          secrets: |-
            ASTRAL_LOCATION=ASTRAL_LOCATION:latest
            SUREPY_AUTH_TOKEN=SUREPY_AUTH_TOKEN:latest
      - name: Trigger deployed function
        run:
          curl ${{ steps.deploy.outputs.url }} --fail-with-body
```


### Log output

```text
2024-08-30T23:08:37.0925784Z Current runner version: '2.319.1'
2024-08-30T23:08:37.0949623Z ##[group]Operating System
2024-08-30T23:08:37.0950230Z Ubuntu
2024-08-30T23:08:37.0950722Z 22.04.4
2024-08-30T23:08:37.0951051Z LTS
2024-08-30T23:08:37.0951364Z ##[endgroup]
2024-08-30T23:08:37.0951840Z ##[group]Runner Image
2024-08-30T23:08:37.0952271Z Image: ubuntu-22.04
2024-08-30T23:08:37.0952636Z Version: 20240825.1.0
2024-08-30T23:08:37.0953696Z Included Software: https://github.com/actions/runner-images/blob/ubuntu22/20240825.1/images/ubuntu/Ubuntu2204-Readme.md
2024-08-30T23:08:37.0955078Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20240825.1
2024-08-30T23:08:37.0955990Z ##[endgroup]
2024-08-30T23:08:37.0956351Z ##[group]Runner Image Provisioner
2024-08-30T23:08:37.0956820Z 2.0.384.1
2024-08-30T23:08:37.0957248Z ##[endgroup]
2024-08-30T23:08:37.0971702Z ##[group]GITHUB_TOKEN Permissions
2024-08-30T23:08:37.0973456Z Contents: read
2024-08-30T23:08:37.0973869Z Metadata: read
2024-08-30T23:08:37.0974555Z ##[endgroup]
2024-08-30T23:08:37.0976633Z Secret source: Actions
2024-08-30T23:08:37.0977409Z Prepare workflow directory
2024-08-30T23:08:37.1590807Z Prepare all required actions
2024-08-30T23:08:37.1745780Z Getting action download info
2024-08-30T23:08:37.3688038Z Download action repository 'actions/checkout@v4.1.7' (SHA:692973e3d937129bcbf40652eb9f2f61becf3332)
2024-08-30T23:08:37.4712831Z Download action repository 'google-github-actions/auth@v2.1.5' (SHA:62cf5bd3e4211a0a0b51f2c6d6a37129d828611d)
2024-08-30T23:08:37.6681188Z Download action repository 'actions/setup-python@v5.2.0' (SHA:f677139bbe7f9c59b41e40162b753c062f5d49a3)
2024-08-30T23:08:37.9521451Z Download action repository 'abatilo/actions-poetry@v3.0.0' (SHA:7b6d33e44b4f08d7021a1dee3c044e9c253d6439)
2024-08-30T23:08:38.1600981Z Download action repository 'google-github-actions/deploy-cloud-functions@v3.0.5' (SHA:a2164ef2fe5dcc9c1f49bb5297133b5ba5920af8)
2024-08-30T23:08:38.7761375Z Complete job name: deploy
2024-08-30T23:08:38.8618319Z ##[group]Run actions/checkout@v4.1.7
2024-08-30T23:08:38.8618709Z with:
2024-08-30T23:08:38.8618981Z   repository: therefromhere/sureflap_gcloud
2024-08-30T23:08:38.8619499Z   token: ***
2024-08-30T23:08:38.8619744Z   ssh-strict: true
2024-08-30T23:08:38.8619996Z   ssh-user: git
2024-08-30T23:08:38.8620248Z   persist-credentials: true
2024-08-30T23:08:38.8620524Z   clean: true
2024-08-30T23:08:38.8620777Z   sparse-checkout-cone-mode: true
2024-08-30T23:08:38.8621090Z   fetch-depth: 1
2024-08-30T23:08:38.8621315Z   fetch-tags: false
2024-08-30T23:08:38.8621558Z   show-progress: true
2024-08-30T23:08:38.8621808Z   lfs: false
2024-08-30T23:08:38.8622025Z   submodules: false
2024-08-30T23:08:38.8622284Z   set-safe-directory: true
2024-08-30T23:08:38.8622555Z ##[endgroup]
2024-08-30T23:08:39.0838406Z Syncing repository: therefromhere/sureflap_gcloud
2024-08-30T23:08:39.0839978Z ##[group]Getting Git version info
2024-08-30T23:08:39.0840696Z Working directory is '/home/runner/work/sureflap_gcloud/sureflap_gcloud'
2024-08-30T23:08:39.0841436Z [command]/usr/bin/git version
2024-08-30T23:08:39.2410155Z git version 2.46.0
2024-08-30T23:08:39.2436961Z ##[endgroup]
2024-08-30T23:08:39.2451605Z Temporarily overriding HOME='/home/runner/work/_temp/81f64442-5754-4cb9-8f28-b159e9ee942c' before making global git config changes
2024-08-30T23:08:39.2452577Z Adding repository directory to the temporary git global config as a safe directory
2024-08-30T23:08:39.2457212Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/sureflap_gcloud/sureflap_gcloud
2024-08-30T23:08:39.2616111Z Deleting the contents of '/home/runner/work/sureflap_gcloud/sureflap_gcloud'
2024-08-30T23:08:39.2619674Z ##[group]Initializing the repository
2024-08-30T23:08:39.2623971Z [command]/usr/bin/git init /home/runner/work/sureflap_gcloud/sureflap_gcloud
2024-08-30T23:08:39.2991574Z hint: Using 'master' as the name for the initial branch. This default branch name
2024-08-30T23:08:39.2992577Z hint: is subject to change. To configure the initial branch name to use in all
2024-08-30T23:08:39.2993829Z hint: of your new repositories, which will suppress this warning, call:
2024-08-30T23:08:39.2994432Z hint:
2024-08-30T23:08:39.2994895Z hint: 	git config --global init.defaultBranch <name>
2024-08-30T23:08:39.2995401Z hint:
2024-08-30T23:08:39.2995935Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2024-08-30T23:08:39.2996840Z hint: 'development'. The just-created branch can be renamed via this command:
2024-08-30T23:08:39.2997456Z hint:
2024-08-30T23:08:39.2997770Z hint: 	git branch -m <name>
2024-08-30T23:08:39.3105684Z Initialized empty Git repository in /home/runner/work/sureflap_gcloud/sureflap_gcloud/.git/
2024-08-30T23:08:39.3117389Z [command]/usr/bin/git remote add origin https://github.com/therefromhere/sureflap_gcloud
2024-08-30T23:08:39.3215235Z ##[endgroup]
2024-08-30T23:08:39.3216209Z ##[group]Disabling automatic garbage collection
2024-08-30T23:08:39.3220968Z [command]/usr/bin/git config --local gc.auto 0
2024-08-30T23:08:39.3250944Z ##[endgroup]
2024-08-30T23:08:39.3251735Z ##[group]Setting up auth
2024-08-30T23:08:39.3259433Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2024-08-30T23:08:39.3297408Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2024-08-30T23:08:39.5474294Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2024-08-30T23:08:39.5520149Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2024-08-30T23:08:39.5783203Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2024-08-30T23:08:39.5824359Z ##[endgroup]
2024-08-30T23:08:39.5825370Z ##[group]Fetching the repository
2024-08-30T23:08:39.5836036Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --no-recurse-submodules --depth=1 origin +c8d3c8b6d7642aa1ef6834f5ce47faa242400137:refs/remotes/pull/76/merge
2024-08-30T23:08:39.8963877Z From https://github.com/therefromhere/sureflap_gcloud
2024-08-30T23:08:39.8965287Z  * [new ref]         c8d3c8b6d7642aa1ef6834f5ce47faa242400137 -> pull/76/merge
2024-08-30T23:08:39.9069567Z ##[endgroup]
2024-08-30T23:08:39.9070438Z ##[group]Determining the checkout info
2024-08-30T23:08:39.9072861Z ##[endgroup]
2024-08-30T23:08:39.9078314Z [command]/usr/bin/git sparse-checkout disable
2024-08-30T23:08:39.9190881Z [command]/usr/bin/git config --local --unset-all extensions.worktreeConfig
2024-08-30T23:08:39.9219437Z ##[group]Checking out the ref
2024-08-30T23:08:39.9223844Z [command]/usr/bin/git checkout --progress --force refs/remotes/pull/76/merge
2024-08-30T23:08:39.9276371Z Note: switching to 'refs/remotes/pull/76/merge'.
2024-08-30T23:08:39.9277175Z 
2024-08-30T23:08:39.9277972Z You are in 'detached HEAD' state. You can look around, make experimental
2024-08-30T23:08:39.9279152Z changes and commit them, and you can discard any commits you make in this
2024-08-30T23:08:39.9280065Z state without impacting any branches by switching back to a branch.
2024-08-30T23:08:39.9280765Z 
2024-08-30T23:08:39.9281251Z If you want to create a new branch to retain commits you create, you may
2024-08-30T23:08:39.9282545Z do so (now or later) by using -c with the switch command. Example:
2024-08-30T23:08:39.9283217Z 
2024-08-30T23:08:39.9283548Z   git switch -c <new-branch-name>
2024-08-30T23:08:39.9283973Z 
2024-08-30T23:08:39.9284172Z Or undo this operation with:
2024-08-30T23:08:39.9284531Z 
2024-08-30T23:08:39.9284682Z   git switch -
2024-08-30T23:08:39.9284930Z 
2024-08-30T23:08:39.9285416Z Turn off this advice by setting config variable advice.detachedHead to false
2024-08-30T23:08:39.9286154Z 
2024-08-30T23:08:39.9286948Z HEAD is now at c8d3c8b Merge 32378e7e32aaabff439fbc1ba840526408c1415e into 2e92e1aef39fd98b96a79861661a88bafef30653
2024-08-30T23:08:39.9289866Z ##[endgroup]
2024-08-30T23:08:39.9347147Z [command]/usr/bin/git log -1 --format='%H'
2024-08-30T23:08:39.9371315Z 'c8d3c8b6d7642aa1ef6834f5ce47faa242400137'
2024-08-30T23:08:39.9693498Z ##[group]Run google-github-actions/auth@v2.1.5
2024-08-30T23:08:39.9693889Z with:
2024-08-30T23:08:39.9694570Z   workload_identity_provider: projects/800042200122/locations/global/workloadIdentityPools/github-identity-pool/providers/github-provider
2024-08-30T23:08:39.9695448Z   service_account: github-cicd@catflap.iam.gserviceaccount.com
2024-08-30T23:08:39.9695885Z   create_credentials_file: true
2024-08-30T23:08:39.9696201Z   export_environment_variables: true
2024-08-30T23:08:39.9696520Z   universe: googleapis.com
2024-08-30T23:08:39.9696802Z   cleanup_credentials: true
2024-08-30T23:08:39.9697089Z   access_token_lifetime: 3600s
2024-08-30T23:08:39.9697583Z   access_token_scopes: https://www.googleapis.com/auth/cloud-platform
2024-08-30T23:08:39.9698262Z   id_token_include_email: false
2024-08-30T23:08:39.9698670Z ##[endgroup]
2024-08-30T23:08:40.1520335Z Created credentials file at "/home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json"
2024-08-30T23:08:40.2473060Z ##[group]Run actions/setup-python@v5.2.0
2024-08-30T23:08:40.2473534Z with:
2024-08-30T23:08:40.2473958Z   python-version: 3.11
2024-08-30T23:08:40.2474347Z   check-latest: false
2024-08-30T23:08:40.2474835Z   token: ***
2024-08-30T23:08:40.2475228Z   update-environment: true
2024-08-30T23:08:40.2475622Z   allow-prereleases: false
2024-08-30T23:08:40.2476018Z env:
2024-08-30T23:08:40.2476746Z   CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:40.2477814Z   GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:40.2478819Z   GOOGLE_GHA_CREDS_PATH: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:40.2479561Z   CLOUDSDK_CORE_PROJECT: catflap
2024-08-30T23:08:40.2479965Z   CLOUDSDK_PROJECT: catflap
2024-08-30T23:08:40.2480355Z   GCLOUD_PROJECT: catflap
2024-08-30T23:08:40.2480903Z   GCP_PROJECT: catflap
2024-08-30T23:08:40.2481235Z   GOOGLE_CLOUD_PROJECT: catflap
2024-08-30T23:08:40.2481674Z ##[endgroup]
2024-08-30T23:08:40.4028306Z ##[group]Installed versions
2024-08-30T23:08:40.4092058Z Successfully set up CPython (3.11.9)
2024-08-30T23:08:40.4093418Z ##[endgroup]
2024-08-30T23:08:40.4229640Z ##[group]Run abatilo/actions-poetry@v3.0.0
2024-08-30T23:08:40.4230187Z with:
2024-08-30T23:08:40.4230516Z   poetry-version: latest
2024-08-30T23:08:40.4230893Z env:
2024-08-30T23:08:40.4231627Z   CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:40.4232722Z   GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:40.4233707Z   GOOGLE_GHA_CREDS_PATH: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:40.4234469Z   CLOUDSDK_CORE_PROJECT: catflap
2024-08-30T23:08:40.4234889Z   CLOUDSDK_PROJECT: catflap
2024-08-30T23:08:40.4235336Z   GCLOUD_PROJECT: catflap
2024-08-30T23:08:40.4235734Z   GCP_PROJECT: catflap
2024-08-30T23:08:40.4236060Z   GOOGLE_CLOUD_PROJECT: catflap
2024-08-30T23:08:40.4236608Z   pythonLocation: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:40.4237359Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib/pkgconfig
2024-08-30T23:08:40.4238025Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:40.4238608Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:40.4239132Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:40.4239783Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib
2024-08-30T23:08:40.4240286Z ##[endgroup]
2024-08-30T23:08:40.4457074Z ##[group]Run pip install --user pipx
2024-08-30T23:08:40.4457895Z [36;1mpip install --user pipx[0m
2024-08-30T23:08:40.4458291Z [36;1mpipx ensurepath[0m
2024-08-30T23:08:40.5484727Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2024-08-30T23:08:40.5485294Z env:
2024-08-30T23:08:40.5486124Z   CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:40.5487298Z   GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:40.5488815Z   GOOGLE_GHA_CREDS_PATH: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:40.5489539Z   CLOUDSDK_CORE_PROJECT: catflap
2024-08-30T23:08:40.5489952Z   CLOUDSDK_PROJECT: catflap
2024-08-30T23:08:40.5490429Z   GCLOUD_PROJECT: catflap
2024-08-30T23:08:40.5490800Z   GCP_PROJECT: catflap
2024-08-30T23:08:40.5491179Z   GOOGLE_CLOUD_PROJECT: catflap
2024-08-30T23:08:40.5491706Z   pythonLocation: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:40.5492360Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib/pkgconfig
2024-08-30T23:08:40.5492999Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:40.5493629Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:40.5494210Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:40.5494780Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib
2024-08-30T23:08:40.5495311Z ##[endgroup]
2024-08-30T23:08:43.7335154Z Collecting pipx
2024-08-30T23:08:43.7628003Z   Downloading pipx-1.7.1-py3-none-any.whl.metadata (18 kB)
2024-08-30T23:08:43.7997139Z Collecting argcomplete>=1.9.4 (from pipx)
2024-08-30T23:08:43.8075234Z   Downloading argcomplete-3.5.0-py3-none-any.whl.metadata (16 kB)
2024-08-30T23:08:43.8364057Z Collecting packaging>=20 (from pipx)
2024-08-30T23:08:43.8438949Z   Downloading packaging-24.1-py3-none-any.whl.metadata (3.2 kB)
2024-08-30T23:08:43.8701296Z Collecting platformdirs>=2.1 (from pipx)
2024-08-30T23:08:43.8795872Z   Downloading platformdirs-4.2.2-py3-none-any.whl.metadata (11 kB)
2024-08-30T23:08:43.8976766Z Collecting userpath!=1.9,>=1.6 (from pipx)
2024-08-30T23:08:43.9052288Z   Downloading userpath-1.9.2-py3-none-any.whl.metadata (3.0 kB)
2024-08-30T23:08:43.9306413Z Collecting click (from userpath!=1.9,>=1.6->pipx)
2024-08-30T23:08:43.9380041Z   Downloading click-8.1.7-py3-none-any.whl.metadata (3.0 kB)
2024-08-30T23:08:43.9499467Z Downloading pipx-1.7.1-py3-none-any.whl (78 kB)
2024-08-30T23:08:43.9667150Z Downloading argcomplete-3.5.0-py3-none-any.whl (43 kB)
2024-08-30T23:08:43.9763150Z Downloading packaging-24.1-py3-none-any.whl (53 kB)
2024-08-30T23:08:43.9863232Z Downloading platformdirs-4.2.2-py3-none-any.whl (18 kB)
2024-08-30T23:08:43.9981723Z Downloading userpath-1.9.2-py3-none-any.whl (9.1 kB)
2024-08-30T23:08:44.0077166Z Downloading click-8.1.7-py3-none-any.whl (97 kB)
2024-08-30T23:08:44.0518580Z Installing collected packages: platformdirs, packaging, click, argcomplete, userpath, pipx
2024-08-30T23:08:44.2298743Z Successfully installed argcomplete-3.5.0 click-8.1.7 packaging-24.1 pipx-1.7.1 platformdirs-4.2.2 userpath-1.9.2
2024-08-30T23:08:44.5218387Z /home/runner/.local/bin is already in PATH.
2024-08-30T23:08:44.5219242Z /opt/pipx_bin is already in PATH.
2024-08-30T23:08:44.5221802Z ⚠️  All pipx binary directories have been appended to PATH. If you are sure
2024-08-30T23:08:44.5223170Z you want to proceed, try again with the '--force' flag.
2024-08-30T23:08:44.5223799Z 
2024-08-30T23:08:44.5223809Z 
2024-08-30T23:08:44.5224166Z Otherwise pipx is ready to go! ✨ 🌟 ✨
2024-08-30T23:08:44.5407376Z ##[group]Run pipx install poetry
2024-08-30T23:08:44.5407814Z [36;1mpipx install poetry[0m
2024-08-30T23:08:44.5473234Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2024-08-30T23:08:44.5473729Z env:
2024-08-30T23:08:44.5474521Z   CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:44.5475880Z   GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:44.5476942Z   GOOGLE_GHA_CREDS_PATH: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:44.5477745Z   CLOUDSDK_CORE_PROJECT: catflap
2024-08-30T23:08:44.5478168Z   CLOUDSDK_PROJECT: catflap
2024-08-30T23:08:44.5478513Z   GCLOUD_PROJECT: catflap
2024-08-30T23:08:44.5478964Z   GCP_PROJECT: catflap
2024-08-30T23:08:44.5479343Z   GOOGLE_CLOUD_PROJECT: catflap
2024-08-30T23:08:44.5479896Z   pythonLocation: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:44.5480557Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib/pkgconfig
2024-08-30T23:08:44.5481160Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:44.5481807Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:44.5482402Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:44.5482995Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib
2024-08-30T23:08:44.5483560Z ##[endgroup]
2024-08-30T23:08:44.6681445Z creating virtual environment...
2024-08-30T23:08:44.8821232Z installing poetry...
2024-08-30T23:08:52.8593877Z done! ✨ 🌟 ✨
2024-08-30T23:08:52.8594646Z   installed package poetry 1.8.3, installed using Python 3.11.9
2024-08-30T23:08:52.8595446Z   These apps are now globally available
2024-08-30T23:08:52.8596236Z     - poetry
2024-08-30T23:08:52.8838417Z ##[group]Run poetry install
2024-08-30T23:08:52.8838972Z [36;1mpoetry install[0m
2024-08-30T23:08:52.8904498Z shell: /usr/bin/bash -e {0}
2024-08-30T23:08:52.8905007Z env:
2024-08-30T23:08:52.8905797Z   CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:52.8906985Z   GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:52.8908035Z   GOOGLE_GHA_CREDS_PATH: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:52.8908697Z   CLOUDSDK_CORE_PROJECT: catflap
2024-08-30T23:08:52.8909171Z   CLOUDSDK_PROJECT: catflap
2024-08-30T23:08:52.8909563Z   GCLOUD_PROJECT: catflap
2024-08-30T23:08:52.8909934Z   GCP_PROJECT: catflap
2024-08-30T23:08:52.8910368Z   GOOGLE_CLOUD_PROJECT: catflap
2024-08-30T23:08:52.8910811Z   pythonLocation: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:52.8911627Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib/pkgconfig
2024-08-30T23:08:52.8912256Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:52.8912817Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:52.8913441Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:52.8913977Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib
2024-08-30T23:08:52.8914460Z ##[endgroup]
2024-08-30T23:08:53.4355135Z Creating virtualenv sureflap-gcloud-7j93eK9t-py3.11 in /home/runner/.cache/pypoetry/virtualenvs
2024-08-30T23:08:53.9257612Z Installing dependencies from lock file
2024-08-30T23:08:54.0811357Z 
2024-08-30T23:08:54.0812775Z Package operations: 71 installs, 0 updates, 0 removals
2024-08-30T23:08:54.0813424Z 
2024-08-30T23:08:54.0817130Z   - Installing protobuf (5.28.0)
2024-08-30T23:08:54.0820495Z   - Installing pyasn1 (0.6.0)
2024-08-30T23:08:54.0823643Z   - Installing pycparser (2.22)
2024-08-30T23:08:54.6137520Z   - Installing cachetools (5.5.0)
2024-08-30T23:08:54.6146004Z   - Installing certifi (2024.8.30)
2024-08-30T23:08:54.6155679Z   - Installing cffi (1.17.0)
2024-08-30T23:08:54.6163630Z   - Installing charset-normalizer (3.3.2)
2024-08-30T23:08:54.6182279Z   - Installing googleapis-common-protos (1.65.0)
2024-08-30T23:08:54.6195609Z   - Installing grpcio (1.66.1)
2024-08-30T23:08:54.6203220Z   - Installing idna (3.8)
2024-08-30T23:08:54.6221537Z   - Installing pyasn1-modules (0.4.0)
2024-08-30T23:08:54.6935956Z   - Installing rsa (4.9)
2024-08-30T23:08:54.6995198Z   - Installing urllib3 (2.2.2)
2024-08-30T23:08:55.0435923Z   - Installing frozenlist (1.4.1)
2024-08-30T23:08:55.0443871Z   - Installing google-auth (2.34.0)
2024-08-30T23:08:55.0452218Z   - Installing grpcio-status (1.66.1)
2024-08-30T23:08:55.0469617Z   - Installing markupsafe (2.1.5)
2024-08-30T23:08:55.0477461Z   - Installing multidict (6.0.5)
2024-08-30T23:08:55.0488579Z   - Installing packaging (24.1)
2024-08-30T23:08:55.0497328Z   - Installing proto-plus (1.24.0)
2024-08-30T23:08:55.0512016Z   - Installing pycares (4.4.0)
2024-08-30T23:08:55.1263644Z   - Installing requests (2.32.3)
2024-08-30T23:08:55.1451753Z   - Installing six (1.16.0)
2024-08-30T23:08:55.1689364Z   - Installing wrapt (1.16.0)
2024-08-30T23:08:55.2056730Z   - Installing zipp (3.20.1)
2024-08-30T23:08:55.2551132Z   - Installing aiodns (3.2.0)
2024-08-30T23:08:55.2560722Z   - Installing aiohappyeyeballs (2.4.0)
2024-08-30T23:08:55.2568850Z   - Installing aiosignal (1.3.1)
2024-08-30T23:08:55.2579365Z   - Installing attrs (24.2.0)
2024-08-30T23:08:55.2587423Z   - Installing blinker (1.8.2)
2024-08-30T23:08:55.2603185Z   - Installing brotli (1.1.0)
2024-08-30T23:08:55.2611603Z   - Installing click (8.1.7)
2024-08-30T23:08:55.2626636Z   - Installing colorama (0.4.6)
2024-08-30T23:08:55.3301016Z   - Installing commonmark (0.9.1)
2024-08-30T23:08:55.3374846Z   - Installing deprecated (1.2.14)
2024-08-30T23:08:55.3390532Z   - Installing deprecation (2.1.0)
2024-08-30T23:08:55.3428503Z   - Installing google-api-core (2.19.2)
2024-08-30T23:08:55.3633271Z   - Installing importlib-metadata (8.4.0)
2024-08-30T23:08:55.3975085Z   - Installing iniconfig (2.0.0)
2024-08-30T23:08:55.4081585Z   - Installing itsdangerous (2.2.0)
2024-08-30T23:08:55.4125942Z   - Installing jinja2 (3.1.4)
2024-08-30T23:08:55.4146031Z   - Installing pluggy (1.5.0)
2024-08-30T23:08:55.4448690Z   - Installing pygments (2.18.0)
2024-08-30T23:08:55.4539425Z   - Installing python-dateutil (2.9.0.post0)
2024-08-30T23:08:55.4733246Z   - Installing werkzeug (3.0.4)
2024-08-30T23:08:55.4925477Z   - Installing yarl (1.9.4)
2024-08-30T23:08:55.7188096Z   - Installing aiohttp (3.10.5)
2024-08-30T23:08:55.7198579Z   - Installing async-timeout (4.0.3)
2024-08-30T23:08:55.7206029Z   - Installing cloudevents (1.11.0)
2024-08-30T23:08:55.7222008Z   - Installing flask (3.0.3)
2024-08-30T23:08:55.7230916Z   - Installing freezegun (1.5.1)
2024-08-30T23:08:55.7241893Z   - Installing google-cloud-appengine-logging (1.4.5)
2024-08-30T23:08:55.7249975Z   - Installing google-cloud-audit-log (0.3.0)
2024-08-30T23:08:55.7265151Z   - Installing google-cloud-core (2.4.1)
2024-08-30T23:08:55.7891403Z   - Installing grpc-google-iam-v1 (0.13.1)
2024-08-30T23:08:55.8094679Z   - Installing gunicorn (23.0.0)
2024-08-30T23:08:55.8162650Z   - Installing mypy-extensions (1.0.0)
2024-08-30T23:08:55.8256421Z   - Installing opentelemetry-api (1.27.0)
2024-08-30T23:08:55.8463052Z   - Installing pathspec (0.12.1)
2024-08-30T23:08:55.8751434Z   - Installing platformdirs (4.2.2)
2024-08-30T23:08:55.8812067Z   - Installing pytest (8.3.2)
2024-08-30T23:08:55.8873537Z   - Installing rich (10.16.2)
2024-08-30T23:08:55.9015153Z   - Installing watchdog (5.0.0)
2024-08-30T23:08:56.0627538Z   - Installing astral (3.2)
2024-08-30T23:08:56.0628711Z   - Installing asyncio (3.4.3)
2024-08-30T23:08:56.0637119Z   - Installing black (24.8.0)
2024-08-30T23:08:56.0661202Z   - Installing functions-framework (3.8.1)
2024-08-30T23:08:56.0669408Z   - Installing google-cloud-logging (3.11.2)
2024-08-30T23:08:56.0677161Z   - Installing pytest-freezegun (0.4.2)
2024-08-30T23:08:56.0690623Z   - Installing pytz (2024.1)
2024-08-30T23:08:56.0701824Z   - Installing surepy (0.9.0 https://github.com/therefromhere/surepy/archive/refs/heads/dst_bug_workaround.zip)
2024-08-30T23:08:56.9853597Z 
2024-08-30T23:08:56.9854479Z Installing the current project: sureflap_gcloud (0.1.0)
2024-08-30T23:08:57.0532677Z ##[group]Run poetry run pytest -v
2024-08-30T23:08:57.0533115Z [36;1mpoetry run pytest -v[0m
2024-08-30T23:08:57.0598902Z shell: /usr/bin/bash -e {0}
2024-08-30T23:08:57.0599386Z env:
2024-08-30T23:08:57.0600103Z   CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:57.0601243Z   GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:57.0602321Z   GOOGLE_GHA_CREDS_PATH: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:57.0603061Z   CLOUDSDK_CORE_PROJECT: catflap
2024-08-30T23:08:57.0603470Z   CLOUDSDK_PROJECT: catflap
2024-08-30T23:08:57.0603921Z   GCLOUD_PROJECT: catflap
2024-08-30T23:08:57.0604305Z   GCP_PROJECT: catflap
2024-08-30T23:08:57.0604648Z   GOOGLE_CLOUD_PROJECT: catflap
2024-08-30T23:08:57.0605220Z   pythonLocation: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:57.0605967Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib/pkgconfig
2024-08-30T23:08:57.0606696Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:57.0607283Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:57.0607829Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:57.0608697Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib
2024-08-30T23:08:57.0609205Z ##[endgroup]
2024-08-30T23:08:58.3106543Z ============================= test session starts ==============================
2024-08-30T23:08:58.3109215Z platform linux -- Python 3.11.9, pytest-8.3.2, pluggy-1.5.0 -- /home/runner/.cache/pypoetry/virtualenvs/sureflap-gcloud-7j93eK9t-py3.11/bin/python
2024-08-30T23:08:58.3111018Z cachedir: .pytest_cache
2024-08-30T23:08:58.3111799Z rootdir: /home/runner/work/sureflap_gcloud/sureflap_gcloud
2024-08-30T23:08:58.3112518Z configfile: pyproject.toml
2024-08-30T23:08:58.3113127Z plugins: freezegun-0.4.2
2024-08-30T23:08:58.6828628Z collecting ... collected 2 items
2024-08-30T23:08:58.6829150Z 
2024-08-30T23:08:58.9759338Z sureflap_gcloud/tests/test_time.py::test_before_start_dst_nz PASSED      [ 50%]
2024-08-30T23:08:58.9833779Z sureflap_gcloud/tests/test_time.py::test_after_start_dst_nz PASSED       [100%]
2024-08-30T23:08:58.9834602Z 
2024-08-30T23:08:58.9834922Z =============================== warnings summary ===============================
2024-08-30T23:08:58.9836753Z ../../../.cache/pypoetry/virtualenvs/sureflap-gcloud-7j93eK9t-py3.11/lib/python3.11/site-packages/pytest_freezegun.py:17
2024-08-30T23:08:58.9838762Z ../../../.cache/pypoetry/virtualenvs/sureflap-gcloud-7j93eK9t-py3.11/lib/python3.11/site-packages/pytest_freezegun.py:17
2024-08-30T23:08:58.9840678Z ../../../.cache/pypoetry/virtualenvs/sureflap-gcloud-7j93eK9t-py3.11/lib/python3.11/site-packages/pytest_freezegun.py:17
2024-08-30T23:08:58.9842789Z ../../../.cache/pypoetry/virtualenvs/sureflap-gcloud-7j93eK9t-py3.11/lib/python3.11/site-packages/pytest_freezegun.py:17
2024-08-30T23:08:58.9844408Z sureflap_gcloud/tests/test_time.py::test_before_start_dst_nz
2024-08-30T23:08:58.9845312Z sureflap_gcloud/tests/test_time.py::test_before_start_dst_nz
2024-08-30T23:08:58.9846315Z sureflap_gcloud/tests/test_time.py::test_after_start_dst_nz
2024-08-30T23:08:58.9847240Z sureflap_gcloud/tests/test_time.py::test_after_start_dst_nz
2024-08-30T23:08:58.9849791Z   /home/runner/.cache/pypoetry/virtualenvs/sureflap-gcloud-7j93eK9t-py3.11/lib/python3.11/site-packages/pytest_freezegun.py:17: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
2024-08-30T23:08:58.9852143Z     if LooseVersion(pytest.__version__) < LooseVersion('3.6.0'):
2024-08-30T23:08:58.9852712Z 
2024-08-30T23:08:58.9853273Z -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
2024-08-30T23:08:58.9854199Z ======================== 2 passed, 8 warnings in 0.67s =========================
2024-08-30T23:08:59.0334447Z ##[group]Run poetry export --output=requirements.txt
2024-08-30T23:08:59.0335070Z [36;1mpoetry export --output=requirements.txt[0m
2024-08-30T23:08:59.0400599Z shell: /usr/bin/bash -e {0}
2024-08-30T23:08:59.0400962Z env:
2024-08-30T23:08:59.0401737Z   CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:59.0402829Z   GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:59.0403839Z   GOOGLE_GHA_CREDS_PATH: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:59.0404606Z   CLOUDSDK_CORE_PROJECT: catflap
2024-08-30T23:08:59.0405025Z   CLOUDSDK_PROJECT: catflap
2024-08-30T23:08:59.0405368Z   GCLOUD_PROJECT: catflap
2024-08-30T23:08:59.0405797Z   GCP_PROJECT: catflap
2024-08-30T23:08:59.0406168Z   GOOGLE_CLOUD_PROJECT: catflap
2024-08-30T23:08:59.0406800Z   pythonLocation: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:59.0407413Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib/pkgconfig
2024-08-30T23:08:59.0408042Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:59.0408958Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:59.0409547Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:59.0410200Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib
2024-08-30T23:08:59.0410702Z ##[endgroup]
2024-08-30T23:08:59.3991806Z Warning: poetry-plugin-export will not be installed by default in a future version of Poetry.
2024-08-30T23:08:59.3995271Z In order to avoid a breaking change and make your automation forward-compatible, please install poetry-plugin-export explicitly. See https://python-poetry.org/docs/plugins/#using-plugins for details on how to install a plugin.
2024-08-30T23:08:59.3998027Z To disable this warning run 'poetry config warnings.export false'.
2024-08-30T23:08:59.6015312Z ##[group]Run google-github-actions/deploy-cloud-functions@v3.0.5
2024-08-30T23:08:59.6015856Z with:
2024-08-30T23:08:59.6016254Z   name: update2
2024-08-30T23:08:59.6016607Z   runtime: python311
2024-08-30T23:08:59.6016974Z   environment: GEN_2
2024-08-30T23:08:59.6017388Z   max_instance_count: 2
2024-08-30T23:08:59.6017722Z   entry_point: update2
2024-08-30T23:08:59.6018119Z   environment_variables: DEVICE_ID=154956
2024-08-30T23:08:59.6018843Z   secrets: ASTRAL_LOCATION=ASTRAL_LOCATION:latest
SUREPY_AUTH_TOKEN=SUREPY_AUTH_TOKEN:latest
2024-08-30T23:08:59.6019490Z   region: us-central1
2024-08-30T23:08:59.6019844Z   universe: googleapis.com
2024-08-30T23:08:59.6020232Z   source_dir: ./
2024-08-30T23:08:59.6020602Z   all_traffic_on_latest_revision: true
2024-08-30T23:08:59.6021026Z   ingress_settings: ALLOW_ALL
2024-08-30T23:08:59.6021470Z   service_timeout: 60s
2024-08-30T23:08:59.6021901Z   vpc_connector_egress_settings: PRIVATE_RANGES_ONLY
2024-08-30T23:08:59.6022512Z   event_trigger_retry: true
2024-08-30T23:08:59.6022900Z env:
2024-08-30T23:08:59.6023522Z   CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:59.6024705Z   GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:59.6025708Z   GOOGLE_GHA_CREDS_PATH: /home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json
2024-08-30T23:08:59.6026373Z   CLOUDSDK_CORE_PROJECT: catflap
2024-08-30T23:08:59.6026853Z   CLOUDSDK_PROJECT: catflap
2024-08-30T23:08:59.6027229Z   GCLOUD_PROJECT: catflap
2024-08-30T23:08:59.6027663Z   GCP_PROJECT: catflap
2024-08-30T23:08:59.6028043Z   GOOGLE_CLOUD_PROJECT: catflap
2024-08-30T23:08:59.6028485Z   pythonLocation: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:59.6029231Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib/pkgconfig
2024-08-30T23:08:59.6029879Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:59.6030454Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:59.6031106Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.9/x64
2024-08-30T23:08:59.6031864Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.9/x64/lib
2024-08-30T23:08:59.6032373Z ##[endgroup]
2024-08-30T23:08:59.7403230Z ##[error]google-github-actions/deploy-cloud-functions failed with: failed to parse secret reference "ASTRAL_LOCATION:latest": unknown format. Secrets should be of the format "projects/p/secrets/s/versions/v".
2024-08-30T23:08:59.7533127Z Post job cleanup.
2024-08-30T23:08:59.8329645Z Removed exported credentials at "/home/runner/work/sureflap_gcloud/sureflap_gcloud/gha-creds-64d5b220005646bf.json".
2024-08-30T23:08:59.8446412Z Post job cleanup.
2024-08-30T23:08:59.9487253Z [command]/usr/bin/git version
2024-08-30T23:08:59.9533494Z git version 2.46.0
2024-08-30T23:08:59.9577027Z Temporarily overriding HOME='/home/runner/work/_temp/30c58e58-5969-4322-ac42-995edcd80338' before making global git config changes
2024-08-30T23:08:59.9578484Z Adding repository directory to the temporary git global config as a safe directory
2024-08-30T23:08:59.9582316Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/sureflap_gcloud/sureflap_gcloud
2024-08-30T23:08:59.9617226Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2024-08-30T23:08:59.9650228Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2024-08-30T23:08:59.9963021Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2024-08-30T23:08:59.9987728Z http.https://github.com/.extraheader
2024-08-30T23:08:59.9999228Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2024-08-30T23:09:00.0033355Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2024-08-30T23:09:00.0489981Z Cleaning up orphan processes
```


### Additional information

_No response_

--- a/README.md
+++ b/README.md
@@ -245,10 +245,10 @@ jobs:
     with:
       secrets: |-
         # As an environment variable:
-        KEY1=secret-key-1:latest
+        KEY1=projects/my-project/secrets/my-secret/versions/latest
 
         # As a volume mount:
-        /secrets/api/key=secret-key-2:latest
+        /secrets/api/key=projects/my-project/secrets/my-secret/versions/123
     ```
 
     This value will only be set if the input is a non-empty value. If a

--- a/action.yml
+++ b/action.yml
@@ -215,7 +215,7 @@ inputs:
     description: |-
       The amount of memory available for the function to use. Allowed values are
       of the format: <number><unit> with allowed units of "k", "M", "G", "Ki",
-      "Mi", "Gi" (e.g 128M, 10Mb, 1024Gib).
+      "Mi", "Gi" (e.g 128M, 10Mb, 1024Gi).
 
       For all generations, the default value is 256MB of memory.
     required: false
@@ -286,10 +286,10 @@ inputs:
       with:
         secrets: |-
           # As an environment variable:
-          KEY1=secret-key-1:latest
+          KEY1=projects/my-project/secrets/my-secret/versions/latest
 
           # As a volume mount:
-          /secrets/api/key=secret-key-2:latest
+          /secrets/api/key=projects/my-project/secrets/my-secret/versions/123
       ```
 
       This value will only be set if the input is a non-empty value. If a

